test/integration/tcp_proxy_integration_test.cc

```diff
@@ -68,6 +69,87 @@ void TcpProxyIntegrationTest::initialize() {
   BaseIntegrationTest::initialize();
 }
 
+// Setup envoy logging access given a file path and a set of logging parameters.
+template <typename LogConfig>
+inline void setListenerAccessLog(ConfigHelper& config_helper, std::string& access_log_path,
```

Comment:
It seems like `params` is only used with strings? If so, can we keep is simple and have params be a string and not template this function